pycmr/granule.py

```diff
@@ -15,6 +16,12 @@ class GranuleQuery(object):
     def __init__(self):
         pass
 
+    def urlEncodeString(self, input):
```

Comment:
This one's minor too, but in python since there isn't an enforced concept of visibility in classes, the standard has been to preface internal/private methods with an underscore in order to hint that it isn't meant to be used by an outside caller.